lib/main.dart

```diff
@@ -165,6 +171,7 @@ class _HomeScreenState extends State<HomeScreen> {
     });
   }
 
+  // Carlo
```

Comment:
Good separation of wide and narrow layouts. Makes it easier to manage complex UI structures.
Good encapsulation of adding, deleting, etc. tasks in this class starting from line 222.